src/sentry/static/sentry/app/views/groupDetails/organization/index.jsx

```diff
@@ -35,4 +34,10 @@ class OrganizationGroupDetails extends React.Component {
   }
 }
 
-export default withOrganization(withGlobalSelection(OrganizationGroupDetails));
+const OrganizationGroupDetailsHoC = withOrganization(
+  withGlobalSelection(OrganizationGroupDetails)
+);
+
+export default function OrganizationGroupDetailsContainer(props) {
+  return <OrganizationGroupDetailsHoC disableLoadFromStore={true} {...props} />;
```

Comment:
Do you need the value here? I thought react would default value-less attributes to `true`?

Reply:
It's not needed, but was being consistent with other props in this component --  we should lint for this